gcloud-java-logging/baseline/src/main/java/com/google/gcloud/logging/spi/v2/MetricsServiceV2Settings.java

```diff
@@ -290,4 +176,170 @@ public Object extractNextToken(ListLogMetricsResponse payload) {
               return payload.getMetricsList();
             }
           };
+
+  public static class Builder extends ServiceApiSettings.Builder {
+    private final ImmutableList<ApiCallSettings.Builder> methodSettingsBuilders;
+
+    private PageStreamingCallSettings.Builder<
+            ListLogMetricsRequest, ListLogMetricsResponse, LogMetric>
+        listLogMetricsSettings;
+    private SimpleCallSettings.Builder<GetLogMetricRequest, LogMetric> getLogMetricSettings;
+    private SimpleCallSettings.Builder<CreateLogMetricRequest, LogMetric> createLogMetricSettings;
+    private SimpleCallSettings.Builder<UpdateLogMetricRequest, LogMetric> updateLogMetricSettings;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'deleteLogMetricSettings' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/mziccard/gcloud-java/file/2388714612/issues/source?bid=3123104&fileBranchId=3228843#l189)